common/test/acceptance/pages/lms/teams.py

```diff
@@ -125,6 +131,7 @@ def browse_teams_for_topic(self, topic_name):
         """
         Show the teams list for `topic_name`.
         """
+        self.wait_for_element_visibility(TEAMS_LINK_CSS, "Waiting for Teams List link to be visible")
```

Comment:
I'm having a hard time adding this comment in the right place (something about how github handles expanding unchanged sections? idk), but a note: click_create_team_link, click_search_team_link, and click_browse_all_teams_link (lines 179, 186, and 193 in the new file, respectively) have *not* been updated to wait for their elements, since they already include logic to handle the element not being present.